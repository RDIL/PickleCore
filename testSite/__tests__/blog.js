import fs from "fs"

describe("blog tests", () => {
    it("outputted the example post", () => {
        expect(fs.existsSync("_site/2019/01/01/test.html")).toBe(true)
    })
    
    it("the post's html is the same as last time", () => {
        expect(fs.readFileSync("_site/2019/01/01/test.html").toString()).toMatchSnapshot()
    })
})
