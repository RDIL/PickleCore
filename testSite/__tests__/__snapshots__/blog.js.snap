// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blog tests the post's html is the same as last time 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
    <head>
        <meta charset=\\"utf-8\\">
<meta http-equiv=\\"X-UA-Compatible\\" content=\\"IE=edge\\">
<meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\">
<meta name=\\"msapplication-TileColor\\" content=\\"#303f9f\\">
<meta name=\\"theme-color\\" content=\\"#303f9f\\">
<!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Test | Example</title>
<meta name=\\"generator\\" content=\\"Jekyll v3.8.7\\" />
<meta property=\\"og:title\\" content=\\"Test\\" />
<meta property=\\"og:locale\\" content=\\"en_US\\" />
<meta name=\\"description\\" content=\\"This is a test. Hello world!!\\" />
<meta property=\\"og:description\\" content=\\"This is a test. Hello world!!\\" />
<link rel=\\"canonical\\" href=\\"https://example.com/2019/01/01/test.html\\" />
<meta property=\\"og:url\\" content=\\"https://example.com/2019/01/01/test.html\\" />
<meta property=\\"og:site_name\\" content=\\"Example\\" />
<meta property=\\"og:type\\" content=\\"article\\" />
<meta property=\\"article:published_time\\" content=\\"2019-01-01T00:00:00+00:00\\" />
<script type=\\"application/ld+json\\">
{\\"headline\\":\\"Test\\",\\"dateModified\\":\\"2019-01-01T00:00:00+00:00\\",\\"datePublished\\":\\"2019-01-01T00:00:00+00:00\\",\\"url\\":\\"https://example.com/2019/01/01/test.html\\",\\"description\\":\\"This is a test. Hello world!!\\",\\"mainEntityOfPage\\":{\\"@type\\":\\"WebPage\\",\\"@id\\":\\"https://example.com/2019/01/01/test.html\\"},\\"@type\\":\\"BlogPosting\\",\\"@context\\":\\"https://schema.org\\"}</script>
<!-- End Jekyll SEO tag -->









    


        <link rel=\\"stylesheet\\" href=\\"/assets/css/page.css\\"></link>
    </head>
    <body>
        
        <main class=\\"content\\">
            <h1>Test</h1>
<section class=\\"post-meta\\"><i class=\\"fas fa-user-edit\\"></i> Author: Administrator
    <br />
    <i class=\\"fas fa-calendar\\"></i> Date: Jan 1, 2019
    
</section>

<hr />
<p>This is a test.
Hello world!!</p>


        </main>
        <link rel=\\"stylesheet\\" href=\\"https://use.fontawesome.com/releases/v5.7.2/css/all.css\\" integrity=\\"sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr\\" crossorigin=\\"anonymous\\">
    </body>
</html>
"
`;
